itest/rust/src/engine_tests/utilities_test.rs

```diff
@@ -10,7 +10,7 @@
 
 use crate::framework::itest;
 
-use godot::builtin::{GString, Variant};
+use godot::{godot_str, builtin::{GString, Variant}};
```

Comment:
The macro should be in `godot::global`, not `godot`, see
https://godot-rust.github.io/docs/gdext/master/godot/global/index.html#macros

This is likely automatically the case when you apply the previous suggestion of moving the `godot_str!` declaration to `print.rs`.

Reply:
How's this? https://github.com/PierceLBrooks/gdext/pull/1117/commits/2446cc86b6816ffe266a95a437f39ca9cec127d5

Reply:
I really meant `print.rs`. While the string is technically not "print", what matters much more is that closely related code is co-located. Having one function in its own file just makes it harder to discover and requires extra navigation if e.g. a refactor across all these macros happens. 

So please keep it in the same file, it's absolutely fine if that is called `print.rs` :slightly_smiling_face: